src/diagram/components/dialog/dialog.scss

```diff
@@ -0,0 +1,52 @@
+.qp-dialog {
+  position: absolute;
+  left: 50px;
+  top: 50px;
+
+  padding: 15px;
+  background-color: white;
+  border-radius: 3px;
+  border: 2px solid #999;
+}
+
+.dialog-content {
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+}
+
+.dialog-row {
+  display: flex;
+  align-items: center;
+  gap: 5px;
+}
+
+.dialog-label {
+  width: 82px;
+  text-align: right;
+  font-size: 12px;
+  font-weight: normal;
+}
+
+.dialog-input {
+  border-radius: 5px;
+  border: 1.5px solid #949494;
+  padding: 3.5px 6px;
+  font-family: PTSans-Narrow, sans-serif;
```

Comment:
I'm curious about this font choice (https://fonts.google.com/specimen/PT+Sans+Narrow). Is it guaranteed that it will be available at runtime?

Reply:
I really don't know the answer to this one. I just used the font Michael had in the spec.

Reply:
If you look at the `index.html` in CLUE, we have lines like these:
```html
    <link href="https://fonts.googleapis.com/css?family=Lato:400,900&display=swap" rel="stylesheet">
    <link href="https://fonts.googleapis.com/css?family=Noto+Sans+Math&display=swap" rel="stylesheet">
    <link href="https://fonts.googleapis.com/css?family=Noto+Sans+Symbols+2&display=swap" rel="stylesheet">
```
According to https://fonts.google.com/specimen/PT+Sans+Narrow, this should be the following for this font:
```html
<link rel="preconnect" href="https://fonts.googleapis.com">
<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
<link href="https://fonts.googleapis.com/css2?family=PT+Sans+Narrow:wght@400;700&display=swap" rel="stylesheet">
```
and that same page says the CSS should be:
```css
font-family: 'PT Sans Narrow', sans-serif;
```

Reply:
This is very useful, thank you. To not hold up this story any longer, I'm going to move ahead without addressing this, but I'll take care of it next time I need to modify quantity playground, which should be very soon.

I notice that quantity playground has an `index.html`, but diagram view doesn't. Should the links you list above be added to quantity playground `index.html`, to `src/diagram/index.ts`, to CLUE's `index.html`, or something else?

Reply:
I'll let @scytacki answer that one as he's more familiar with the use of standalone quantity playground.